tests/Css/StyleTest.php

```diff
@@ -39,6 +39,29 @@ public function testLengthInPt(string $length, ?float $ref_size, $expected): voi
         $this->assertSame($expected, $result);
     }
 
+    public function calcLengthInPtProvider(): array
+    {
+        return [
+          ["calc(100%)", null, 12.0],
+          ["calc(50% - 1pt)", 200, 99.0],
+          ["calc(100)", null, 100.0],
+          ["calc(100% / 3)", 100, 33.333333333333336],
+        ];
+    }
+
+    /**
+     * @dataProvider calcLengthInPtProvider
+     */
+    public function testCalcLengthInPt(string $length, ?float $ref_size, $expected): void
```

Comment:
Since the method is identical to `testCalcLengthInPt`, I would merge both, plus the providers; just separate the `calc` tests with a blank line in the data provider for clarity.